hm-module.nix

```
@@ -0,0 +1,79 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}: let
+  inherit (lib) types;
+  cfg = config.programs.webcord;
+in {
+  options = {
+    programs.webcord = {
+      enable = lib.mkEnableOption cfg.package.meta.description;
+
+      package = lib.mkOption {
+        type = types.package;
+        default = pkgs.webcord;
+        description = lib.mdDoc ''
+          Package to use for WebCord configuration.
+        '';
+        example = lib.literalExpression ''
+          pkgs.webcord.overrideAttrs (old: {
```

```suggestion
          inputs.webcord.packages.${pkgs.system}.default.override {
```
Not really necessary, just giving the users a hint about how to approach overriding flake packages.
Also it's `override`, not `overrideAttrs`.